src/main/java/ploopy/Deadline.java

```diff
@@ -0,0 +1,21 @@
+package ploopy;
+
+public class Deadline extends Task {
```

Comment:
Perhaps you can have separation of the files into different packages as you deem fit.